### PR TITLE
Modify S5693: Add syntax highlighting

### PR DESCRIPTION
**Pull request by loris-s-sonarsource**

### CODE REVIEW

**Review by hendrik-buchwald-sonarsource — APPROVED** [review]
LGTM, you might want to remove the spaces before the language though. Functionally, it probably does not make a difference, but as far as I know we don't do that in the other rules.

File: rules/S5693/javascript/rule.adoc [hunk 1]
```diff
@@ -8,6 +8,7 @@ include::../recommended.adoc[]
 
 https://www.npmjs.com/package/formidable[formidable] file upload module:
 
+[source, javascript]
```
Comment (loris-s-sonarsource) [inline rules/S5693/javascript/rule.adoc:11]:
```suggestion
[source,javascript]
```

File: rules/S5693/javascript/rule.adoc [hunk 2]
```diff
@@ -17,6 +18,7 @@ const formDefault = new Formidable(); // Sensitive, the default value is 200MB
 
 https://www.npmjs.com/package/multer[multer] (Express.js middleware) file upload module:
 
+[source, javascript]
```
Comment (loris-s-sonarsource) [inline rules/S5693/javascript/rule.adoc:21]:
```suggestion
[source,javascript]
```

File: rules/S5693/javascript/rule.adoc [hunk 3]
```diff
@@ -32,6 +34,7 @@ let diskUploadUnlimited = multer({ // Sensitive: the default value is no limit
 
 https://www.npmjs.com/package/body-parser[body-parser] module:
 
+[source, javascript]
```
Comment (loris-s-sonarsource) [inline rules/S5693/javascript/rule.adoc:37]:
```suggestion
[source,javascript]
```

File: rules/S5693/php/rule.adoc [hunk 4]
```diff
@@ -8,7 +8,7 @@ include::../recommended.adoc[]
 
 For https://symfony.com/doc/current/reference/constraints/File.html#maxsize[Symfony Constraints]:
 
-
+[source, php]
```
Comment (loris-s-sonarsource) [inline rules/S5693/php/rule.adoc:11]:
```suggestion
[source,php]
```

File: rules/S5693/php/rule.adoc [hunk 5]
```diff
@@ -26,7 +26,7 @@ class TestEntity
 
 For https://laravel.com/docs/8.x/validation#rule-max[Laravel Validator]:
 
-
+[source, php]
```
Comment (loris-s-sonarsource) [inline rules/S5693/php/rule.adoc:29]:
```suggestion
[source,php]
```

File: rules/S5693/vbnet/rule.adoc [hunk 6]
```diff
@@ -6,6 +6,7 @@ include::../recommended.adoc[]
 
 == Sensitive Code Example
 
+[source, vbnet]
```
Comment (loris-s-sonarsource) [inline rules/S5693/vbnet/rule.adoc:9]:
```suggestion
[source,vbnet]
```

**Pull request merged**